com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ The version number for this package has increased due to a version update of a r
 - Fixed a regression where the precision was changed. [case 1313942](https://issuetracker.unity3d.com/issues/urp-shader-precision-is-reduced-to-half-when-scriptablerenderfeature-class-is-in-the-project)
 - Fixed issue causing missing shaders on DirectX 11 feature level 10 GPUs. [case 1278390](https://issuetracker.unity3d.com/product/unity/issues/guid/1278390/)
 
+### Added
+- Added SpeedTree8 shadergraph for universal. Since it behaves differently from existing ST8 shader, it is not hooked up or set to be default yet.
```

Comment:
Proposed:
Added a SpeedTree 8 Shader Graph but did not make it accessible to users or set it as the default. Because URP doesn't yet support per-material culling, this Shader Graph does not yet behave in the same way as the existing handwritten SpeedTree 8 shader for URP.

Reply:
It is technically accessible for users. When the universal package is included in their project, it also includes the new shader graph. However, importing and upgrading ST8 assets still uses the original handwritten ST8 shader. 

So, how about:
"Added a SpeedTree 8 Shader  Graph but did not set it as the default when importing or upgrading Speed Tree 8 assets. Because URP doesn't yet support per-material culling, this Shader Graph does not yet behave in the same way as the existing handwritten SpeedTree 8 shader for URP."

Reply:
Swap out "when" in the first sentence with "for" and  then this is good.